### PR TITLE
Publish Letter from Executive Director

### PR DESCRIPTION
**Pull request by kmomar**

### CODE REVIEW

File: _posts/2020/07/2020-07-09-july-2020-letter-from-the-executive-director.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+authors: ["Kari L. Jordan"]
+teaser: "Global Capacity Building for Inclusive Computational Communities "
```
Comment (ErinBecker) [inline _posts/2020/07/2020-07-09-july-2020-letter-from-the-executive-director.md]: trailing space

File: _posts/2020/07/2020-07-09-july-2020-letter-from-the-executive-director.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+title: "Letter from The Carpentries Executive Director, July 2020"
+date: 2020-07-09
+time: "00:09:00"
+tags: ["Community"]
+---
+Dear Carpentries Community,<br />
+
+On June 12 our Core Team participated in the first [Reading and Reflection Day](https://carpentries.org/blog/2020/06/reading-reflection-days/). On that day I spent time digging into resources on building and sustaining non-profit projects. I learned many things in the time I spent reading, but what leapt from the pages were strategies to move beyond survival and into thriving.
```
Comment (ErinBecker) [inline _posts/2020/07/2020-07-09-july-2020-letter-from-the-executive-director.md]: 12 June

File: _posts/2020/07/2020-07-09-july-2020-letter-from-the-executive-director.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Now, don’t get me wrong. The Carpentries is not only surviving - we are thriving! Your feedback and continued engagement has allowed us to establish processes for piloting our workshops online, we’ve increased our number of trained instructors by 29%, and we are seeing a 20% increase in workshops. That combined with support from [CZI and the Moore Foundation](https://carpentries.org/blog/2019/11/czi-moore-grant/) and [Mozilla’s Open Source Support (MOSS)](https://carpentries.org/blog/2020/02/mozilla-grant-announcement/) program, the road ahead includes scaling instructor training, curriculum development, community development, and our equity, inclusion, and accessibility (EIA) efforts. There’s lots to do - and I’m here for all of it!
```
Comment (ErinBecker) [inline _posts/2020/07/2020-07-09-july-2020-letter-from-the-executive-director.md]: Make it clear increase is compared to 2019

File: _posts/2020/07/2020-07-09-july-2020-letter-from-the-executive-director.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (ErinBecker) [inline _posts/2020/07/2020-07-09-july-2020-letter-from-the-executive-director.md:1]: Check for contractions. 🙂

**Comment by ErinBecker** [comment]
@kariljordan - looks great! I left a few formatting comments, and a reminder to read through for contractions (I'm finally learning! @serahrono 🙂 )

I also wanted to note the colloquialisms, which might not translate well to our international community - "kicking off", "jam packed", "digging into". Maybe "inaugurating" , "action filled", "immersed in"? Just a few suggestions, please feel free to ignore!

**Comment by kariljordan** [comment]
Thanks @ErinBecker! [Updated](https://github.com/carpentries/carpentries.org/pull/809/commits/e78dfcdb413147012442a38c65d8c49083f4fe98)

**Pull request merged**